lib/operations/mongo_client_ops.js

```diff
@@ -439,10 +439,14 @@ function createTopology(mongoClient, topologyType, options, callback) {
     }
 
     const MongoClient = loadClient();
-    const connectionString =
-      os.platform() === 'win32'
-        ? 'mongodb://localhost:27020/?serverSelectionTimeoutMS=1000'
-        : 'mongodb://%2Ftmp%2Fmongocryptd.sock/?serverSelectionTimeoutMS=1000';
+    let connectionString;
+    if (options.autoEncryption.extraOptions && options.autoEncryption.extraOptions.mongocryptURI) {
```

Comment:
I just want to do a sanity check to make sure that `autoEncryption.extraOptions.mongocryptURI` will be set on `options` here and we won't have to check `mongoClient.s.options`. Since we don't have the tests for FLE yet, maybe we can make a ticket for this test and add it later?

Reply:
Sure - making the ticket right now. Just to clarify - do you mean ensuring that the `options` parameter passed into `createTopology` on line 388 has the property `autoEncryption.extraOptions`?

Reply:
I mean making sure that a `mongocryptURI` set on `MongoClient` will be passed to the `options` parameter on line 388.

Reply:
ticket created: https://jira.mongodb.org/browse/NODE-2047